File: src/Services/WareHouse/WareHouse.Domain/Entity/Unit.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace WareHouse.Domain.Entity
{
    public partial class Unit:BaseEntity
    {
        public Unit()
        {
            BeginningWareHouses = new HashSet<BeginningWareHouse>();
            InwardDetails = new HashSet<InwardDetail>();
            OutwardDetails = new HashSet<OutwardDetail>();
            WareHouseItemUnits = new HashSet<WareHouseItemUnit>();
            WareHouseItems = new HashSet<WareHouseItem>();
            WareHouseLimits = new HashSet<WareHouseLimit>();
        }

        public string UnitName { get; set; }
        public bool Inactive { get; set; }

        public virtual ICollection<BeginningWareHouse> BeginningWareHouses { get; set; }
        public virtual ICollection<InwardDetail> InwardDetails { get; set; }
        public virtual ICollection<OutwardDetail> OutwardDetails { get; set; }
        public virtual ICollection<WareHouseItemUnit> WareHouseItemUnits { get; set; }
        public virtual ICollection<WareHouseItem> WareHouseItems { get; set; }
        public virtual ICollection<WareHouseLimit> WareHouseLimits { get; set; }
    }
}

File: src/WebClient/WareHouse/src/app/entity/Unit.ts
import { BaseEntity } from "./BaseEntity";
import { BeginningWareHouse } from "./BeginningWareHouse";
import { Inward } from "./Inward";
import { InwardDetail } from "./InwardDetail";
import { OutwardDetail } from "./OutwardDetail";
import { WareHouseItem } from "./WareHouseItem";
import { WareHouseItemUnit } from "./WareHouseItemUnit";
import { WareHouseLimit } from "./WareHouseLimit";
export interface Unit extends BaseEntity {
    unitName: string;
    inactive: boolean;
    beginningWareHouses: BeginningWareHouse[];
    inwardDetails: InwardDetail[];
    outwardDetails: OutwardDetail[];
    wareHouseItemUnits: WareHouseItemUnit[];
    wareHouseItems: WareHouseItem[];
    wareHouseLimits: WareHouseLimit[];
}


